### PR TITLE
New entry, text and images CSS [Edit] CSS Position: bottom() #4071

### PR DESCRIPTION
**Pull request by Aukse1976**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/position/terms/bottom/bottom.md [hunk 1]
```diff
@@ -56,3 +60,22 @@ Set the position of `.box` element `40px` from the elements bottom edge.
   bottom: 40px;
 }
 ```
+![Shows the use of CSS bottom attribute with element position set to relative](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-relative-bottom.png)
+
+The blue box `position: relative` is positioned 40px above its default position, represented by the dashed box within the containing element `position: relative`.
```
Comment (avdhoottt) [inline content/css/concepts/position/terms/bottom/bottom.md:65]:
```suggestion
The blue box `position: relative` is positioned `40px` above its default position, represented by the dashed box within the containing element `position: relative`.
```

File: content/css/concepts/position/terms/bottom/bottom.md [hunk 2]
```diff
@@ -42,6 +42,10 @@ Set the position of `.box` element `40px` from the bottom of the nearest relativ
   bottom: 40px;
 }
 ```
+![Shows the use of CSS bottom attribute with element position set to absolute](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-absolute-bottom.png)
+
+The blue box `position: absolute` is positioned 40px from the bottom edge of its containing element `position: relative`.
```
Comment (avdhoottt) [inline content/css/concepts/position/terms/bottom/bottom.md:47]:
```suggestion
The blue box `position: absolute` is positioned `40px` from the bottom edge of its containing element `position: relative`.
```

File: content/css/concepts/position/terms/bottom/bottom.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ CatalogContent:
   - 'paths/full-stack-engineer-career-path'
 ---
 
-Specifies the distance of an element from the bottom of its current location if the element itself has `position: relative` declared or from the bottom of its nearest ancestor when the ancestor has `position: relative` and the element has `position: absolute` declared.
+Specifies the distance of an element from the bottom of its current location if the element itself has `position: relative` declared or from the bottom of its nearest ancestor when the ancestor has `position: relative` and the element has `position: absolute` declared. `position: fixed` positioned an element relative to the viewport. It takes the element out of the normal flow and remains fixed, not affected by scrolling.
```
Comment (avdhoottt) [inline content/css/concepts/position/terms/bottom/bottom.md:16]:
```suggestion
Specifies the distance of an element from the bottom of its current location if the element itself has `position: relative` declared or from the bottom of its nearest ancestor when the ancestor has `position: relative` and the element has `position: absolute` declared. `position: fixed` positions an element relative to the viewport. It takes the element out of the normal flow and remains fixed, not affected by scrolling.
```

File: content/css/concepts/position/terms/bottom/bottom.md [hunk 4]
```diff
@@ -56,3 +60,22 @@ Set the position of `.box` element `40px` from the elements bottom edge.
...
+.box {
+  height: 100px;
+  width: 100px;
+  background-color: blue;
+  position: fixed;
+  bottom: 40px;
+}
+```
+
+The blue box `position: fixed` is taken out of the normal document flow and its position with bottom property sets its distance 40px above the bottom edge of the viewport. It doesn't move when the page is scrolled.
```
Comment (avdhoottt) [inline content/css/concepts/position/terms/bottom/bottom.md:81]:
```suggestion
The blue box `position: fixed` is taken out of the normal document flow and its position with the bottom property sets its distance 40px above the bottom edge of the viewport. It doesn't move when the page is scrolled.
```

**Review by avdhoottt — APPROVED** [review]
Great images! PR Looks good for the second review.

**Review by dakshdeepHERE — APPROVED** [review]
LGTM 🚀

**Comment by Aukse1976** [comment]
> LGTM 🚀

Thanks for the review and your approval! 🙌

**Pull request merged**